--- ./xclip.el
+++ ./xclip.el
@@ -77,6 +77,7 @@
    (and (executable-find "xclip") 'xclip)
    (and (executable-find "xsel") 'xsel)
    (and (executable-find "wl-copy") 'wl-copy) ;github.com/bugaevc/wl-clipboard
+   (and (executable-find "sopaste") 'soclip)
    (and (executable-find "termux-clipboard-get") 'termux-clipboard-get)
    (and (fboundp 'x-create-frame) (getenv "DISPLAY") 'emacs)
    (and (eq system-type 'gnu/linux) ;FIXME: How do we detect WSL?
@@ -91,6 +92,7 @@
           (const :tag "X11: xclip" xclip)
           (const :tag "X11: xsel" xsel)
           (const :tag "Wayland: wl-copy" wl-copy)
+          (const :tag "soclip: soclip" soclip)
           (const :tag "Termux: termux-clipboard-get/set" termux-clipboard-get)
           (const :tag "WSL: clip.exe/powershell.exe" powershell)
           (const :tag "X11: Emacs" emacs)))
@@ -147,6 +149,11 @@
                  (apply #'start-process
                         "wl-copy" nil xclip-program
                         (if (memq type '(primary PRIMARY)) '("-p")))))
+              (`soclip
+               (start-process
+                "soclip" nil (replace-regexp-in-string
+                              "\\(.*\\)sopaste" "\\1socopy"
+                              xclip-program 'fixedcase)))
               (`termux-clipboard-get
                (when (memq type '(clipboard CLIPBOARD))
                  (start-process "termux-clipboard-set" nil
@@ -203,6 +210,8 @@
                   ;;    automatically enabled for non-text content types and
                   ;;    when using the --watch mode.
                   "-n" (if (memq type '(primary PRIMARY)) '("-p")))))
+        (`soclip
+         (call-process xclip-program nil standard-output nil))
         (`termux-clipboard-get
          (when (memq type '(clipboard CLIPBOARD))
            (call-process xclip-program nil standard-output nil)))
